_maps/readme.md

```diff
@@ -4,29 +4,25 @@
 
 The list of items that are modular requiring adding/modifying to /tg/ maps:
 
-- Positronic Alert Consoles (inside Robotics area)
-- Command Vendor (inside Meeting area)
```

Comment:
I don't really think that moving this to the loadout is the good move to make, I'd rather we just add on to the garment bags in the heads' lockers instead, if we're not going to use the command vendor anymore. Would be relatively easy to do modularly, without having the maintenance cost that the Command vendor has.

Reply:
It'll be moved to the loadout menu and or into lockers, this single change I always forget about, it's not something we need to keep around, or we can just move it to the interlink